_source/_posts/2025-06-16-sohail-pathan-intro.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: blog_post
+title: "Superheroes, Startups, and Security: Sohail's Path to Developer Advocacy at Okta"
+author: sohail-pathan
+by: advocate
+communities: [javascript,security,java,mobile]
+description: "Join Sohail Pathan's inspiring journey from Android dev to Okta's Sr. Developer Advocate. Discover his passion for tech, DevRel, APIs, and community building."
+tags: [advocacy, javascript, kotlin, okta]
+image: blog/sohail-pathan-intro/sohail-pathan-social-image.png
+type: awareness
+---
+
+### **Hello okta community 👋**
+
+Let me tell you a story—a story of dreams, passion, and the continuous pursuit of curiosity. My name is Sohail Pathan, and I'm thrilled to join Okta as a Senior Developer Advocate. 
+
+It all started in Nagpur, a quaint city nestled in central Ind`ia. As a child, I eagerly peered through my window, convinced superheroes were real and that someday Superman himself would swing by. While superheroes never appeared, another hero soon took their place—**technology**.
+
+School never captivated me much, but the computer lab was a magical place. Recognizing this spark, my parents gifted me my first desktop computer—a moment that forever changed my life. A formal education in Computer Science followed, where I fell in love with Java and the vibrant Android ecosystem. When [Google introduced Kotlin support in 2017](https://android-developers.googleblog.com/2017/05/android-announces-support-for-kotlin.html), weekends turned into coding marathons, building Android apps for everyday use.
+
+### **My professional journey: from an app developer to a developer advocate**
+
+My professional path started at an EdTech startup. Here, my first role as an Android developer unfolded with both excitement and nervous anticipation. I vividly remember my early days, meticulously building and refining app features, driven by the genuine joy of seeing students interact with the app. On the go, my involvement quickly expanded beyond just coding. I found myself drawn to answering student queries on forums and guiding them through their learning journeys.
+
+<div style="text-align: center;">
```

Comment:
You should be able to add `{: .center-image}` to the image. i've used `<cite>` for this. Take a look at 

https://github.com/oktadev/okta-blog/blob/main/_source/_posts/2022-07-06-spa-web-security.md

Reply:
Done - For all the images.